### ISSUE TITLE
Panaroo core gene alignment issue

### ISSUE THREAD

**Issue by hkaspersen** [NorwegianVeterinaryInstitute/ALPPACA#120]
@karinlag detected that there were no correspondence between the detected core genes and the resulting alignment from Panaroo. An alignment was produced even though zero core genes were identified. We therefore have to figure out what actually goes into the alignment. We have asked Gerry on the µbioinfo slack about this.

**Comment by hkaspersen** [comment]
It seems from the documentation that the `core_gene_alignment.aln` should be based on the genes included within the parameters specified with `--core_threshold`, as defined [here](https://gthlab.au/panaroo/#/gettingstarted/output?id=core_gene_alignmentaln). Perhaps this is an issue with the discrepancy between the stats file and the output?

**Comment by evezeyl** [comment]
@hkaspersen do we have info on where @karinlag has done the tests and command that was run ? 
best to try to reproduce and find the error with these data maybe ?

**Comment by hkaspersen** [comment]
@evezeyl yes that sounds like something we need to test. @karinlag could you provide us the data so we can replicate?

**Comment by hkaspersen** [comment]
Reply from Gerry on Panaroo:

> The --core_threshold parameter controls what goes into the core alignment.
We have also added additional filtering steps which generates the core_gene_alignment_filtered.aln file which generally gives much better phylogenies if that is what you're after.
This filtering can be controlled with the  --core_entropy_filter parameter.
I think deciding what constitutes a core gene is often very context-specific. For example, if you're just building a tree, then an arbitrary threshold might be fine. However, if you're trying to determine whether a gene is found in all genomes of a species, this is likely to be insufficient.

**Comment by karinlag** [comment]
@hkaspersen where are we here? Have we moved forward on this yet?

**Comment by hkaspersen** [comment]
Using the PhiX genome as a test:
Panaroo identified seven (7) genes as core genes.
Core gene alignment length: 4950 bp

Total length of individual core genes:
1: 170
2: 458
3: 116 
4: 1283
5: 527
6: 986
7: 1403

Sum: 4943 bp, which is very close to the length of the core gene alignment.
Running a test on real data with the same setup to check.

**Comment by hkaspersen** [comment]
Real teset using five K. pn. genomes:
Core genes: 4497
Alignment length: 4.228.156 bp
Sum of core gene lengths: 4.228.156 bp

Conclusion: Correspondence between core gene alignment length and reported core gene numbers.
